HIP/hip-0000-adding-pause–feature-to-HTS.md

```diff
@@ -0,0 +1,82 @@
+- hip: [TBD]
+- title: Pause feature on Hedera Token Service
+- author: chung@xfers.com, junhao@xfers.com, zhikai@xfers.com
+- type: Standards Track
+- category: Application
+- status: Draft
+- created: 2021-08-09
+- discussions-to: 
+- updated: 2021-08-31
+
+## **Abstract**
+
+Stablecoin tokens issued on various protocols usually have a pause function included as part of their smart contract. This allows the token issuer to pause the whole smart contract for contract upgrading, emergencies and other scenarios. This function is currently not available in the Hedera Token Service. Therefore, we propose to establish a standard format for pause feature in HTS. 
+
+## **Motivation**
+
+There are several reasons that would justify the need for a pause feature;
+
+1. Compliance with regulatory frameworks across jurisdictions in the case of regulated token issuance (e.g. stablecoins)
+2. Compliance with law enforcement requests
+3. Parity with functionalities offered by other blockchain protocols in the case of multichain asset issuance.
+
+We propose to establish a standard for HTS where a pause feature can be included for tokens deployed via HTS. This will be opt-in and allows token issuers the flexibility of including or excluding the pause functionality based on the token issuers' requirements. 
+
+## **Specification**
+
+To be Completed.
```

Comment:
Should be completed before merged.

Reply:
Noted. Would finalize that before the end of next Monday, SGT